sdks/java/io/kinesis/src/test/java/org/apache/beam/sdk/io/kinesis/ShardReadersPoolTest.java

```diff
@@ -76,7 +75,7 @@ public void setUp() throws TransientKinesisException {
     when(thirdIterator.getShardId()).thenReturn("shard3");
     when(fourthIterator.getShardId()).thenReturn("shard4");
     KinesisReaderCheckpoint checkpoint = new KinesisReaderCheckpoint(
-        Arrays.asList(firstCheckpoint, secondCheckpoint));
+        asList(firstCheckpoint, secondCheckpoint));
```

Comment:
`Arrays.asList` is already a hack and not readable - the method is for converting an array to a list, but it is used here to define a new list "literal", so to speak. But programmers seem to know the pattern so it is not so bad. Removing `Arrays` from the beginning makes it less clear that this hack is being used. These should instead be `ImmutableList.of` (I'm assuming they are all used immutably).